phlib/include/apiimport.h

```diff
@@ -229,7 +229,7 @@ typedef VOID (WINAPI* _UnsubscribeServiceChangeNotifications)(
     _In_ PSC_NOTIFICATION_REGISTRATION pSubscription
     );
 
-#define PH_DECLARE_IMPORT(Name) _##Name Name##_Import(VOID)
+#define PH_DECLARE_IMPORT(Name) _##Name NTAPI Name##_Import(VOID)
```

Comment:
These types don't have calling conventions.

Reply:
well my VS2022 begs to differ without that fix i can not build a x86 version of Task Explorer